mod.yaml

```diff
@@ -198,32 +202,32 @@ LobbyDefaults:
 	TechLevel: Unrestricted
 
 ChromeMetrics:
-	ra2:metrics.yaml
+	ra2|metrics.yaml
```

Comment:
I think that can be done in another pr (obviously has to, as this one is merged :p).